src/app/pages/details/desktop-view/desktop-view.js

```diff
@@ -78,34 +73,22 @@ function StubUntilSeen({active, ...JLParams}) {
     );
 }
 
-export default function DesktopView({onContentChange}) {
+function setCardBackground(isShowingCards) {
+    const el = document.querySelector('.details-page');
+
+    el?.classList[isShowingCards ? 'add' : 'remove']('card-background');
```

Comment:
To make this more "react-y" we should probably use state and Context to render this in details instead of manipulating the dom